clang/include/clang/Driver/Options.td

```diff
@@ -5899,8 +5899,10 @@ def print_enabled_extensions : Flag<["-", "--"], "print-enabled-extensions">,
   HelpText<"Print the extensions enabled by the given target and -march/-mcpu options."
            " (AArch64 and RISC-V only)">,
   MarshallingInfoFlag<FrontendOpts<"PrintEnabledExtensions">>;
-def : Flag<["-"], "mcpu=help">, Alias<print_supported_cpus>;
-def : Flag<["-"], "mtune=help">, Alias<print_supported_cpus>;
+def : Flag<["-"], "mcpu=help">, Alias<print_supported_cpus>,
+  Visibility<[ClangOption, CC1Option, CLOption, FlangOption, FC1Option]>;
+def : Flag<["-"], "mtune=help">, Alias<print_supported_cpus>,
+  Visibility<[ClangOption, CC1Option, CLOption, FlangOption, FC1Option]>;
```

Comment:
I'd be tempted to extract these to some documented block, similar to https://github.com/llvm/llvm-project/blob/4086ead63c7e0b56b3b07873117bb7ad7a02d41c/clang/include/clang/Driver/Options.td#L3672-L3691.

Reply:
That's a good idea. I have moved the options into such a block. I have begun adding support for `-print-supported-extensions` and `-print-enabled-extensions` as well which can eventually be added to the block.